mlflow/models/evaluation/base.py

```diff
@@ -1278,7 +1291,13 @@ def evaluate(
           specified, unless the ``evaluator_config`` option **log_model_explainability** is
           explicitly set to ``True``.
 
-    :param model: A pyfunc model instance, or a URI referring to such a model.
+    :param model: One of the following:
+
+                  - A pyfunc model instance
```

Comment:
@liangz1 does this actually work, or is it outdated documentation? We *should* make it work :)

Reply:
Yes, added a test `test_evaluate_with_loaded_pyfunc_model`.